packages/@react-spectrum/menu/src/MenuTrigger.tsx

```diff
@@ -59,9 +59,10 @@ function MenuTrigger(props: SpectrumMenuTriggerProps, ref: DOMRef<HTMLElement>)
     onClose: state.close,
     closeOnSelect,
     autoFocus: state.focusStrategy || true,
-    UNSAFE_style: {
-      width: isMobile ? '100%' : undefined
-    },
+    UNSAFE_style: isMobile ? {
+      width: '100%',
+      'max-height': 'inherit'
```

Comment:
To fix the tests
```suggestion
      'maxHeight': 'inherit'
```
cuz useStyleProps uses `maxHeight` instead of `max-height`